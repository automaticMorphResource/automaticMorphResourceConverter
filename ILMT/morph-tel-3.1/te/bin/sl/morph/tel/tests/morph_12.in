1	kAgA	unk
1	..	unk	
1	3	unk	
1	lakRala	unk	
1	caxaraMyoVkka	unk	
1	kilomItarla	unk	
1	meraku	unk	
1	vyApiMci	unk	
1	uMde	unk	
1	gret	unk	
1	bAriyar	unk	
1	rIP	unk	
1	lo	unk	
1	cinna	unk	
1	cinna	unk	
1	xIvulu	unk	
1	eVnno	unk	
1	unnAyi	unk	
1	.	unk	
