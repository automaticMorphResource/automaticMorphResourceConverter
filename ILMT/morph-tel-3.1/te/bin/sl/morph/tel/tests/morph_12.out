1	kAgA	unk	<fs af='kAgA,avy,,,,,,'>|<fs af='kAgu,n,,sg,,,0,0_A'>|<fs af='kAgu,v,any,any,any,,i_A,i_A'>|<fs af='avvu,v,any,any,any,,a,a_gA'>
2	..	unk	<fs af='&dot;,punc,,,,,,'>
3	3	unk	<fs af='3,num,,,,,,'>
4	lakRala	unk	<fs af='lakRa,n,,pl,,o,ti,ti'>
5	caxaraMyoVkka	unk	<fs af='caxaraM,n,,sg,,,yoVkka,yoVkka'>
6	kilomItarla	unk	<fs af='kilomItar,n,,pl,,o,ti,ti'>
7	meraku	unk	<fs af='meraku,avy,,,,,0,0_avy'>|<fs af='mera,n,,sg,,,ki,ki'>
8	vyApiMci	unk	<fs af='vyApiMcu,v,any,any,any,,i,i'>
9	uMde	unk	<fs af='uMdu,v,any,any,any,,e,e'>|<fs af='uMdu,v,any,any,any,,i,i_e'>
10	gret	unk	<fs af='gret,n,,sg,,d,0,0'>|<fs af='gret,n,,sg,,o,ti,ti'>
11	bAriyar	unk	<fs af='bAriyar,n,,sg,,d,0,0'>|<fs af='bAriyar,n,,sg,,o,ti,ti'>
12	rIP	unk	<fs af='rIP,n,,sg,,d,0,0'>|<fs af='rIP,n,,sg,,o,ti,ti'>
13	lo	unk	<fs af='lo,avy,,,,,,'>|<fs af='la,avy,,,,,o,o_avy'>
14	cinna	unk	<fs af='cinna,n,,sg,,d,0,0'>|<fs af='cinna,n,,sg,,o,ti,ti'>|<fs af='cinna,adj,,,,,0,0_adj'>|<fs af='cinnaM,n,,sg,,o,ti,ti'>
15	cinna	unk	<fs af='cinna,n,,sg,,d,0,0'>|<fs af='cinna,n,,sg,,o,ti,ti'>|<fs af='cinna,adj,,,,,0,0_adj'>|<fs af='cinnaM,n,,sg,,o,ti,ti'>
16	xIvulu	unk	<fs af='xIvi,n,,pl,,d,0,0'>
17	eVnno	unk	<fs af='eVnno,avy,,,,,,'>|<fs af='eVnnu,v,any,any,any,,i,i_o'>|<fs af='eVMwa,n,,pl,,,0,0_o'>
18	unnAyi	unk	<fs af='uMdu,v,n,pl,3,,A,A'>
19	.	unk	<fs af='&dot;,punc,,,,,,'>
